File: LeetCode-Programs/Shuffle String.cpp
class Solution {
public:
    string restoreString(string s, vector<int>& indices) {
        string str="";
        int temp=0;
        while(temp<indices.size()){
            for(int i=0;i<indices.size();i++){
                if(indices[i]==temp){
                    str+=s[i];
                    break;
                }
            }
            temp++;
        }
        return str;
    }
};

File: LeetCode-Programs/Shuffle String.py
class Solution:
    def restoreString(self, s: str, indices: List[int]) -> str:
        string=""
        temp=0
        while temp<len(indices):
            for i in range(len(indices)):
                if temp==indices[i]:
                    string+=s[i]
            temp+=1
        return string
